static/welcome.html.sample

```diff
@@ -1,43 +1,42 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="utf-8">
-    <link rel="stylesheet" href="style/base.css" type="text/css" />
-</head>
-<body style="margin: 0">
-
-    <div class="jumbotron">
-        <div class="container">
-            <h2>Hello, <strong>Galaxy</strong> is running!</h2>
-            To customize this page edit <code>static/welcome.html</code>
-
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheet" href="style/base.css" type="text/css" />
+    </head>
+    <body style="margin: 0">
+        <div class="jumbotron">
+            <div class="container">
+                <h2>Hello, <strong>Galaxy</strong> is running!</h2>
+                To customize this page edit <code>static/welcome.html</code>
+                <br>
+                <a target="_blank" href="https://wiki.galaxyproject.org/Admin/Config" class="btn btn-primary btn-lg">Configuring Galaxy »</a>
+                <a target="_blank" href="https://wiki.galaxyproject.org/Admin/Tools/AddToolFromToolShedTutorial" class="btn btn-primary btn-lg">Installing Tools »</a>
+            </div>
             <br>
-
-            <a target="_blank" href="https://wiki.galaxyproject.org/Admin/Config" class="btn btn-primary btn-lg">Configuring Galaxy »</a>
-            <a target="_blank" href="https://wiki.galaxyproject.org/Admin/Tools/AddToolFromToolShedTutorial" class="btn btn-primary btn-lg">Installing Tools »</a>
-
+            <div class="container">
+                <p>Take an interactive tour:
+                    <a target="_parent" href="/tours/core.galaxy_ui" class="btn btn-default btn-xs">Galaxy UI</a>
```

Comment:
Drop the leading slash, maybe?  Otherwise I think this fails when there's a prefix, right?